### PR TITLE
[AA-1007] Use azure artifact feed

### PR DESCRIPTION
**Pull request by andonyns**
- Use prerelease version of AppCommon with changes to use Azure Artifacts
- Change URLs to use azure artifacts

### CODE REVIEW

File: EdFi.Suite3.Installer.AdminApp/prep-installer-package.ps1 [hunk 1]
```diff
@@ -14,7 +14,7 @@ $ErrorActionPreference = "Stop"
 Push-Location $PackageDirectory
 
 $dependencyVersions = @{
-    AppCommon = "1.1.0"
+    AppCommon = "1.1.0-pre1138"
```
Comment (stephenfuqua) [inline EdFi.Suite3.Installer.AdminApp/prep-installer-package.ps1:17]: Please make sure we have a Tracker ticket for release preparation tasks, and that one of the acceptance criteria is updating this script to the full release.

Separately, we need to change the version number on AppCommon - we have already released 1.1.0, so it doesn't make sense to have new pre-releases of 1.1.0 😆 . Need to bump to 1.2.0. I will look into that, as I'm already working on a ticket that includes an update to AppCommon.

Comment (stephenfuqua) [reply EdFi.Suite3.Installer.AdminApp/prep-installer-package.ps1:17]: 1.2.0 update: done

Comment (stephenfuqua) [reply EdFi.Suite3.Installer.AdminApp/prep-installer-package.ps1:17]: By the way, `1.1.0-pre1138` is the build for my TSINSTALL-88 ticket, which has not yet been reviewed. So this ticket should probably wait for merge until TSINSTALL-88 is done, which hopefully will be this morning.

Comment (andonyns) [reply EdFi.Suite3.Installer.AdminApp/prep-installer-package.ps1:17]: I created https://tracker.ed-fi.org/browse/AA-1218 to track the package update as one of the final steps before the release

**Review by CSR2017 — APPROVED** [review]

**Pull request merged**